assessment_software/src/app/path-display/path-display.component.html

```diff
@@ -0,0 +1,5 @@
+<a *ngFor="let path of pathDisplay; index as i;"
+[href]="links[i]"
+[ngClass]="{pathSkin: i !== 0, first: i === 0}"
```

Comment:
this is very smart. I see that you are really getting comfortable with angular. 👌